app/kubernetes/helpers/namespaceHelper.js

```diff
@@ -1,21 +1,30 @@
 import _ from 'lodash-es';
-import angular from 'angular';
 
-class KubernetesNamespaceHelper {
-  /* @ngInject */
-  constructor(KUBERNETES_SYSTEM_NAMESPACES, KUBERNETES_DEFAULT_NAMESPACE) {
-    this.KUBERNETES_SYSTEM_NAMESPACES = KUBERNETES_SYSTEM_NAMESPACES;
-    this.KUBERNETES_DEFAULT_NAMESPACE = KUBERNETES_DEFAULT_NAMESPACE;
+import { KubernetesNamespace, KUBERNETES_DEFAULT_NAMESPACE, KUBERNETES_DEFAULT_SYSTEM_NAMESPACES } from 'Kubernetes/models/namespace/models';
+import KubernetesNamespaceStore from 'Kubernetes/store/namespace';
+
+export default class KubernetesNamespaceHelper {
+  static isSystemNamespace(namespace) {
+    let ns = namespace;
+    if (typeof ns !== 'string') {
```

Comment:
a bit fragile, why not just make sure you pass a namespace string?

Reply:
Would require to rewrite calls in 16 files and I was trying to avoid that, but that's probably better yes